types/abci.pb.go

```diff
@@ -44,7 +44,7 @@ type TxResponse struct {
 	// non-deterministic.
 	RawLog string `protobuf:"bytes,6,opt,name=raw_log,json=rawLog,proto3" json:"raw_log,omitempty"`
 	// The output of the application's logger (typed). May be non-deterministic.
-	Logs ABCIMessageLogs `protobuf:"bytes,7,rep,name=logs,proto3,castrepeated=ABCIMessageLogs" json:"logs"`
```

Comment:
is this correct? If it is then my debugging for how to fix this issue was time not spent well 😭  lol

Reply:
Ah, it seems that `(gogoproto.castrepeated)` is not taken into account.

castrepeated is a custom feature added on our own fork of gogoproto, maybe that broke when using protoc 3.13.0 instead of 3.11.2. @anilCSE Do you have an idea here?

We could also just downgrade to force protoc 3.11.2 inside `proto-tools-installer.sh`.

Reply:
it looks like this breaks the build and a lot of tests

Reply:
I tried `make proto-gen` and there's a complete diff. I have
```
$ protoc --version
libprotoc 3.11.2
```

Reply:
I think we should stick to `3.11.2`

Reply:
I just ran `make proto-gen` with 3.13.0, and there's no diff (except the forever random staking.pb.go bytes). I would propose for everyone to upgrade to 3.13.0.

Reply:
I am on 3.11.2 (not sure why), and this makes a diff with the cast-repeated stuff, which breaks the build. Not sure why the script didn't update. Maybe cuz I locally installed this ages ago?

Reply:
I believe this script skips the installation if you already have protoc/gocosmos installed, even if the wrong version is installed. an improvement to this script would be to check versions too.

In the meantime:
- you can remove the protoc & gocosmos binaries manually, and run this command again.
- or wait for Marko's docker image, which is a 10x better solution anyways